x/relationships/keeper/keeper_test.go

```diff
@@ -9,24 +9,29 @@ import (
 func (suite *KeeperTestSuite) TestKeeper_StoreRelationship() {
 	tests := []struct {
 		name                string
-		storedRelationships []sdk.AccAddress
+		storedRelationships types.Relationships
 		user                sdk.AccAddress
-		receiver            sdk.AccAddress
+		relationship        types.Relationship
 		expErr              error
 	}{
 		{
-			name:                "already existent relationship returns error",
-			storedRelationships: []sdk.AccAddress{suite.testData.otherUser},
-			user:                suite.testData.user,
-			receiver:            suite.testData.otherUser,
-			expErr:              fmt.Errorf("relationship already exists with %s", suite.testData.otherUser),
+			name: "already existent relationship returns error",
+			storedRelationships: types.Relationships{
+				types.NewRelationship(suite.testData.otherUser,
+					"4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e"),
+			},
+			user: suite.testData.user,
+			relationship: types.NewRelationship(suite.testData.otherUser,
+				"4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e"),
+			expErr: fmt.Errorf("relationship already exists with %s", suite.testData.otherUser),
 		},
 		{
 			name:                "relationship added correctly",
 			storedRelationships: nil,
 			user:                suite.testData.user,
-			receiver:            suite.testData.otherUser,
-			expErr:              nil,
+			relationship: types.NewRelationship(suite.testData.otherUser,
+				"4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e"),
+			expErr: nil,
 		},
 	}
```

Comment:
Can we add two tests to make sure that adding a new relationship with:

- a different subspace
- a different receiver

work properly if another relationship already exists?